app/helpers/models_helper.rb

```diff
@@ -40,6 +40,6 @@ def jws_online_logo
   end
 
   def show_copasi_button?
-    Seek::Config.copasi_enabled && (@display_model.policy.access_type == Policy::ALL_USERS) && @display_model.is_copasi_supported?
+    Seek::Config.copasi_enabled && @display_model.is_copasi_supported? && @display_model.is_downloadable? && @display_model.can_download?(current_user)
```

Comment:
this duplicates and could reuse `@display_model.can_run_copasi?`

Reply:
`can_run_copasi?` is defined for the "model" model, can not be called by @display_model ( versioning model)